GestureHandler.js

```diff
@@ -444,6 +444,20 @@ class BaseButton extends React.Component {
     this._lastActive = false;
   }
 
+  componentDidMount() {
```

Comment:
Imho we should do it only with ‘RawButton’. I see no need for displaying this warning for every component which can be wrapped with ‘NativeViewGestureHandler’